### PR TITLE
Add logging functions to faasr_client_tools.R

### PR DESCRIPTION
**Pull request by nanmu08**
Add logging functions to faasr_client_tool.R

after calling faasr function: faasr_invoke <- faasr(json_path="payload.json", env=".env")

by calling faasr_invoke$get_log_df() will generate a txt file to list all log files

by calling faasr_invoke$get_log("a_uuid_string") to download log files in that uuid folder

by calling faasr_invoke$delete_log("a_uuid_string") to delete that uuid folder

by calling faasr_invoke$delete_log_date("2024-04-15") , when pass a date as argument, will delete all files before that date.

### CODE REVIEW

File: R/faasr_client_tools.R [hunk 1]
```diff
@@ -539,3 +539,294 @@ faasr_unset_workflow_timer <- function(target=NULL,...){
...
+
+      continuation_token <- check_log_file$NextContinuationToken
+    } else {
+      break
+    }
+  }
+
+  # Format dates and create data frame if any data exists
+  if (length(uuids) > 0) {
+    dates <- as.POSIXct(dates, format="%Y-%m-%d %H:%M:%S", tz="")
```
Comment (spark0510) [inline R/faasr_client_tools.R]: `dates <- as.POSIXct(dates, format="%Y-%m-%d %H:%M:%S", tz="")`
This code makes following error:

`Error in as.POSIXct.numeric(dates, format = "%Y-%m-%d %H:%M:%S", tz = "") : 
  'origin' must be supplied`

**Comment by spark0510** [comment]
Checked May 20:
It works well! Thank you!

**Comment by nanmu08** [comment]
update faasr_client_tool.R for fixing “origin” error and time correction(for example: a div.txt last modified date is April 15, 2024, 21:46:58 , but it shows "April 16, 2024, 01:46:58"). Tested in local enviornment and Docker container enviornment.

**Comment by spark0510** [comment]
It's working okay - retrieve time for EDT

**Pull request merged**